easybuild/easyconfigs/g/Gaussian/Gaussian-g16.a03-AVX2.eb

```diff
@@ -0,0 +1,46 @@
+# easyconfig file for Gaussian 16 Rev. A.03 - AVX2
+easyblock = 'PackedBinary'
+
+name = 'Gaussian'
+version = 'g16.a03'
+versionsuffix = '-AVX2'
+
+description = """Gaussian is a computer program for computational quantum chemistry
+that includes various methods for electronic structure calculations."""
+homepage = "http://www.gaussian.com/"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# license needed, download manually
+sources = ['E6B-113X.tbz']
+checksums = ['50aa2c7fd7912da7f77745407b5fdadc7aa7d6b78bb14733d67a246d0a7cbd0f']
+
+postinstallcmds = ['chgrp -R %(namelower)s %(installdir)s/g16']
```

Comment:
This is prone to fail and not really necessary unless you don't have a site license, in which case it's probably better to do that step outside of the easyconfig.

See #4247 for an alternative.

Reply:
In addition the name of the group is very much site-specific anyway...